yamscripts.yml

```diff
@@ -55,6 +55,8 @@ scripts:
     locales: pontoon-to-json --src locales --dest system-addon/data
     # Copy over all of the system-addon directory
     copy: cpx "system-addon/**/{,.}*" ../mozilla-central/browser/extensions/activity-stream
+    copyPingCentre: cpx "system-addon/lib/PingCentre.jsm" ../mozilla-central/browser/modules
+    rmDupePingCentre: rimraf "../mozilla-central/browser/extensions/activity-stream/lib/PingCentre.jsm"
```

Comment:
Instead of doing this, I think it might be a better idea to put `PingCenter` in its own folder at the top level `i.e. ping-centre/PingCentre.jsm`